### PR TITLE
install vagrant and vagrant-libvirt from the copr

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: roles/vagrant/tasks/main.yml [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
   get_url:
-    url: https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
-    dest: /etc/yum.repos.d/hashicorp.repo
+    url: https://copr.fedorainfracloud.org/coprs/pvalena/rubygems/repo/epel-{{ ansible_distribution_major_version }}/pvalena-rubygems-epel-{{ ansible_distribution_major_version }}.repo
+    dest: /etc/yum.repos.d/pvalena-rubygems.repo
+
+- name: 'install pvalena/vagrant copr'
+  get_url:
+    url: https://copr.fedorainfracloud.org/coprs/pvalena/vagrant/repo/epel-{{ ansible_distribution_major_version }}/pvalena-vagrant-epel-{{ ansible_distribution_major_version }}.repo
+    dest: /etc/yum.repos.d/pvalena-vagrant.repo
```
Comment (ekohl) [inline roles/vagrant/tasks/main.yml:10]: Thoughts about using https://docs.ansible.com/ansible/latest/collections/community/general/copr_module.html?

File: roles/vagrant/tasks/main.yml [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
 
 - name: 'install vagrant'
   yum:
```
Comment (ekohl) [inline roles/vagrant/tasks/main.yml:13]: Just a thought: should this be `package` instead of `yum`?

Comment (evgeni) [reply roles/vagrant/tasks/main.yml:13]: given the rest won't work on non-EL, and there is no real difference between `yum` and `dnf`, "yes, but it's just cosmetics"

File: roles/vagrant/tasks/main.yml (hunk 1 above)
Comment (evgeni) [reply roles/vagrant/tasks/main.yml:10]: I prefer to stay away from `community.general`

Comment (ekohl) [reply roles/vagrant/tasks/main.yml:10]: Fair enough.

**Review by ekohl — APPROVED** [review]

**Pull request merged**